apps/galaxy/src/modules/Orders/OpenOrders/OpenOrdersList.tsx

```diff
@@ -42,16 +67,44 @@ const OpenOrdersList = () => {
     }),
   ];
 
+  const Submit = () => {
+    const Orders = checkedItems
+      .map((checked, index) => {
+        if (checked) return index;
+      })
+      .filter((checked) => checked !== undefined);
```

Comment:
Maybe filter `false` first, so that the order of the logic would makes more sense (at least to me, I would filter out the falsy first then map out their indexes)